src/coreComponents/mesh/SurfaceElementRegion.cpp

```diff
@@ -185,10 +178,27 @@ localIndex SurfaceElementRegion::addToFractureMesh( real64 const time_np1,
     }
   }
 
+  return rval;
+}
+
+localIndex SurfaceElementRegion::addToFractureMesh( real64 const time_np1,
+                                                    FaceManager const * const faceManager,
+                                                    ArrayOfArraysView< localIndex const >  const & originalFaceToEdgeMap,
+                                                    localIndex const faceIndices[2] )
+{
+  localIndex const kfe = this->addToSurfaceMesh( faceManager, originalFaceToEdgeMap, faceIndices );
+
+  FaceElementSubRegion & subRegion = this->getUniqueSubRegion< FaceElementSubRegion >();
+  arrayView1d< real64 > const ruptureTime = subRegion.getField< fields::ruptureTime >();
+  ruptureTime( kfe ) = time_np1;
+
+  SurfaceElementSubRegion::EdgeMapType & edgeMap = subRegion.edgeList();
   // Fill the connectivity between FaceElement entries. This is essentially a copy of the
   // edgesToFaces map, but with differing offsets.
-  for( auto const & edge : connectedEdges )
+  for( localIndex a = 0; a < edgeMap.sizeOfArray( kfe ); ++a )
```

Comment:
This needs to be identical to the previous definition, which is a loop over `connectedEdges`, which is set using `originalFaceToEdgeMap` rather than `subRegion.edgeList`.

Reply:
Thank you @rrsettgast, I made it identical to the previous implementation.